R/sgdnet.R

```diff
@@ -376,6 +376,9 @@ sgdnet.default <- function(x,
                            ncol = n_penalties,
                            dimnames = list(variable_names, path_names),
                            sparse = TRUE)
+
+    df <- colSums(as.matrix(beta != 0))
```

Comment:
This might slow (converting the sparse matrix `beta` to a dense matrix) Can we use Matrix::colSums here instead and avoid the conversion?